.ebextensions/yarn.config

```diff
@@ -0,0 +1,8 @@
+packages:
+  rpm:
+    nodesource: https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
+  yum:
+    nodejs: []
+container_commands:
```

Comment:
I don't think this needs to be container commands, but probably fine.

`commands` happens before the new app bundle is downloaded, `container_commands` happens after.